marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/MasteringController.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.controllers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.hub.dataservices.MasteringService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(value = "/api/mastering")
```

Comment:
@fsnow  @ryanjdew  Do you anticipate having other endpoints specific to the concept of mastering? I'm wondering if these collections are specific to matching or merging and thus should be in the Matching or Merging step controller.

Reply:
This will be used for populating the defaults to the merge step, so it could go under merging. I can't think of anything at the moment that couldn't be split between matching or merging.

Reply:
We don't have a MergingController in that directory, so it would be a rename of this one.

Reply:
@fsnow I think this should go into MergingStepController then, as it's specific to a merging step.

Reply:
done